### PR TITLE
rectangles benchmark

### PR DESCRIPTION
**Pull request by ayakayorihiro**
New benchmark for bril for CS6120 Lesson 2 tasks; this benchmark takes the side values of two rectangles, and computes the difference in area between the two rectangles.

### CODE REVIEW

**Comment by sampsyo** [comment]
Looks perfect; thanks!!

File: benchmarks/rectangles-area-difference.bril [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ARGS: 5 10 6 13
+@main (x1 : int, y1: int, x2: int, y2 : int) {
+      a1: int = call @area x1 y1;
+      a2: int = call @area x2 y2;
+      res: int = sub a1 a2;
+      a1_bigger: boolean = gt a1 a2;
```
Comment (Pat-Lafon) [inline benchmarks/rectangles-area-difference.bril:6]: Did you mean: `boolean` -> `bool`?

Comment (ayakayorihiro) [reply benchmarks/rectangles-area-difference.bril:6]: Ahh sorry, it seems like I missed the syntax a bit. But weirdly enough `boolean` also works? (I tested both ways locally) I'll make a PR with the fix, thanks for pointing this out!

Comment (Pat-Lafon) [reply benchmarks/rectangles-area-difference.bril:6]: Yeah it's just #155. It's technically fine for `brili` but I've been trying to keep `brilirs` working on all of the benchmarks and it more strictly follows the specification.

**Pull request merged**